content/en/docs/instrumentation/net/_index.md

```diff
@@ -23,7 +23,7 @@ language:
 
 | Tracing | Metrics | Logging |
 | ------- | ------- | ------- |
-| 1.0    | Alpha   | Beta    |
+| Stable  | Stable  | Stable  |
```

Comment:
Though  ILogger integration is stable, I'd put an "*" here, and indicate that the OTLP Log Exporter is not stable, to avoid any confusion (and incorrectly assume that OTLP Exporter for Logs is also stable)

Reply:
Updated.